parser/Builder.cc

```diff
@@ -391,7 +391,8 @@ class Builder::Impl {
 
         core::NameRef method;
         if (selector == nullptr) {
-            method = core::Names::bang();
+            // when the selector is missing, this is a use of the `call` method.
+            method = core::Names::call();
```

Comment:
What the heck lol why was this ever `bang()` 🤔

Reply:
Yeah..